blender_addons/io_md5_exporter/io_export_md5.py

```diff
@@ -801,26 +807,35 @@ def treat_bone(b, parent = None):
         for mesh in range (1, len(meshes)):
           for submesh in meshes[mesh].submeshes:
             submesh.bindtomesh(meshes[0])
+      checkPath(modelFilePath)#python does not create directory just the file. This function added to create t eh directory

```

Comment:
Comment should not be necesary once the method has a proper name. Also  there is a typo in it that you could fix in the methods documentation.